tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
-    "compilerOptions": {
-        "target": "ES6",
-        "module": "commonjs",
-        "outDir": "dist",
-        "rootDir": "src",
-        "skipLibCheck": true,
-        "strict": true,
-        "esModuleInterop": true,
-        "forceConsistentCasingInFileNames": true
-    }
-}
\ No newline at end of file
+  "compilerOptions": {
+    "target": "ES6",
+    "module": "commonjs",
+    "outDir": "dist",
+    "rootDir": "src",
+    "skipLibCheck": true,
+    "strict": false,
```

Comment:
why changed to false?

Reply:
Because it was too strict :(( Too many errors with the old code when I was refactoring so I just threw in the towel. We can definitely change this later on (will have to do some rewrites again to make sure TypeScript is happy).

Reply:
Oke, fair enough. I'll open an issue for it so that it doesnt get forgotten later